### ISSUE TITLE
Can't load plugin 

### ISSUE THREAD

**Issue by Sunstroke98** [MohrJonas/obsidian-ocr#51]
Hi,

I tried installing the plugin through the community plugin tab directly on obsidian.
I then downloaded tesseract as well as imagemagick from the links provided on the plugin page
I added them to the path using the tutorial provided as well.
However, when i try to enable the plugin i get the message:

![image](https://github.com/MohrJonas/obsidian-ocr/assets/156959076/3267a3d3-e51f-4819-b1b8-a4c77b70115f)

I tried running the debugging console and it gives me the following message:

![image](https://github.com/MohrJonas/obsidian-ocr/assets/156959076/473dc5b6-6e78-4e90-a3ed-7f03ea36db5c)

I'm guessing i didnt correctly add the executables to the path but i cant figure out what to change, could you help out?

**Comment by FranticPanic** [comment]
Receiving the same error as shown above after installing the dependencies and adding them to the system path.

**Comment by gavinwright-engr** [comment]
Same issue happening for me.

**Comment by fredjoe7** [comment]
same issue for me ... just installed plugin and tried to enable it and got this error

**Comment by MohrJonas** [comment]
Thank you for your issue. Can you please create a log file by enabling file logging in the settings and rerunning the steps that lead to the issue. After that you should have a log file in your vault. Please upload it to GitHub gists and link it here.

**Comment by mjelp0147854** [comment]
Does anyone sent you the logfile @MohrJonas? I've got the same issue both with the automatic and the manuel installation process in Windows ...

P.S.: I think the idea behind your plug-in has great potential. Thank you! :)

**Comment by Wingout** [comment]
@MohrJonas Tried installing today using the recommended flow on windows 10 and facing same issue, exe are in path however getting the absolute path error, i will try and log the install and get back to you if i can work through it, plugin looks great from the outside!

**Comment by MohrJonas** [comment]
Thank you for you kind words. Getting the log and (hopefully) finally resolving this issue would be great 😁

**Comment by planetClaire** [comment]
About the log file, how do we enable file logging in Obsidian settings?  I can't see it anywhere...  I notice no-one else has linked a file to this thread yet.  I'm also having the same problem.

Here's the console log shown in the screenshot anyway: https://gist.github.com/planetClaire/84c7435eb9902825a4c56944961ce9ce

> Thank you for your issue. Can you please create a log file by enabling file logging in the settings and rerunning the steps that lead to the issue. After that you should have a log file in your vault. Please upload it to GitHub gists and link it here.

**Comment by planetClaire** [comment]
> About the log file, how do we enable file logging in Obsidian settings?  I can't see it anywhere...  I notice no-one else has linked a file to this thread yet.  I'm also having the same problem.
> 
> 
> 
> Here's the console log shown in the screenshot anyway: https://gist.github.com/planetClaire/84c7435eb9902825a4c56944961ce9ce
> 
> 
> 
> > Thank you for your issue. Can you please create a log file by enabling file logging in the settings and rerunning the steps that lead to the issue. After that you should have a log file in your vault. Please upload it to GitHub gists and link it here.
> 
> 
> 
> 

I see from another issue that the logging is in the plugin options, but the options aren't available until it's enabled.

**Comment by MohrJonas** [comment]
Yes, you're correct that the settings are only available when the plugin loads successfully. This, however is also valuable information, because now we know that the error happens before loading the settings.

**Comment by planetClaire** [comment]
> Yes, you're correct that the settings are only available when the plugin loads successfully. This, however is also valuable information, because now we know that the error happens before loading the settings.

Thanks for your reply.  Just FYI, both tesseract and imagick are in my path.

**Comment by Sergeybond97** [comment]
I followed line numbers from error in console
![image](https://github.com/MohrJonas/obsidian-ocr/assets/43493893/4cf235cc-6f14-443c-8996-414aad8b69f1)

At line 64610 in main.js is function `external_node_path_namespaceObject.dirname` with path to some local developer's directory
And in line 64774 same thing

I replaced calls to those functions with empty string and plugin start to work

![image](https://github.com/MohrJonas/obsidian-ocr/assets/43493893/cb01ebd2-4cdb-4c37-b74c-7890e5e14cf2)

**Comment by py9mrg** [comment]
> I followed line numbers from error in console ![image](https://private-user-images.githubusercontent.com/43493893/339020787-4cf235cc-6f14-443c-8996-414aad8b69f1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgzODIyNjAsIm5iZiI6MTcxODM4MTk2MCwicGF0aCI6Ii80MzQ5Mzg5My8zMzkwMjA3ODctNGNmMjM1Y2MtNmYxNC00NDNjLTg5OTYtNDE0YWFkOGI2OWYxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE0VDE2MTkyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFjZDU3MGViMGY0NTM0ZjAzZjExM2UyYzdiOGM5ZTk3MTQwMDBhZGVhYjVlZWY0MzNiNjM5MTMwMjhiY2JhZGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.QT6WmLPRb6W6YDqSl49zrlnpgxeVfYPN4iCngBwEEYI)
> 
> At line 64610 in main.js is function `external_node_path_namespaceObject.dirname` with path to some local developer's directory And in line 64774 same thing
> 
> I replaced calls to those functions with empty string and plugin start to work
> 
> ![image](https://private-user-images.githubusercontent.com/43493893/339022198-cb01ebd2-4cdb-4c37-b74c-7890e5e14cf2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgzODIyNjAsIm5iZiI6MTcxODM4MTk2MCwicGF0aCI6Ii80MzQ5Mzg5My8zMzkwMjIxOTgtY2IwMWViZDItNGNkYi00YzM3LWI3NGMtNzg5MGU1ZTE0Y2YyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE0VDE2MTkyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJmYzBhY2Y4OTc1NjI1NTdjNTM1MTMwOGIwNWNhYTE5NWNkMGJiZjkwMjMzNDE3Y2YyMzA4ODI3YzgyNGIwZDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.iHxJo-qh7tdvxgTkJOnhHxU8dArRLcN-3jCUohIU5DI)

You hero.

**Comment by ian-loyola** [comment]
> I followed line numbers from error in console ![image](https://github.com/MohrJonas/obsidian-ocr/assets/43493893/4cf235cc-6f14-443c-8996-414aad8b69f1)
> 
> At line 64610 in main.js is function `external_node_path_namespaceObject.dirname` with path to some local developer's directory And in line 64774 same thing
> 
> I replaced calls to those functions with empty string and plugin start to work
> 
> ![image](https://github.com/MohrJonas/obsidian-ocr/assets/43493893/cb01ebd2-4cdb-4c37-b74c-7890e5e14cf2)

This worked for me! Thank you

**Comment by kungfupurpp** [comment]
> I followed line numbers from error in console ![image](https://github.com/MohrJonas/obsidian-ocr/assets/43493893/4cf235cc-6f14-443c-8996-414aad8b69f1)
> 
> At line 64610 in main.js is function `external_node_path_namespaceObject.dirname` with path to some local developer's directory And in line 64774 same thing
> 
> I replaced calls to those functions with empty string and plugin start to work
> 
> ![image](https://github.com/MohrJonas/obsidian-ocr/assets/43493893/cb01ebd2-4cdb-4c37-b74c-7890e5e14cf2)

can confirm as of 7/28/25 this still works
